tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/ContextManager.scala

```diff
@@ -0,0 +1,21 @@
+package me.shadaj.scalapy.tensorflow
+
+import me.shadaj.scalapy.py
+
+object ContextManager {
```

Comment:
Hmm maybe rename to just `Context`? As in, companion object for `Context` trait. `...Manager` is a poor way to name classes, `Manager`s are usually indicator of unclear responsibility :/

Reply:
`Context` is a trait that the classes implement.
But calling this class `ContextManager` was intentional because thats how `with` is called in python. 


> Of all of the most commonly used Python constructs, `context managers`
https://jeffknupp.com/blog/2016/03/07/python-with-context-managers/
itp

Reply:
Ok, let's leave as is then